src/components/organisms/UiList/UiList.vue

```diff
@@ -32,16 +32,47 @@
           </slot>
         </UiListItem>
       </template>
+      <!-- @slot Use this slot to replace button -->
+      <UiListItemButton v-if="hasButton">
+        <template
+          v-for="(_, name) in itemsToRender"
+          #[name]="data"
+        >
+          <slot
+            v-bind="data"
+            :name="name"
+          />
+        </template>
+      </UiListItemButton>
     </slot>
+    <template v-if="hasLoader">
```

Comment:
I checked this loader, and it never shows before component content, because it is the last element, and there is no `v-else` to replace the loader.
So you should add loader as the first element and the rest of the component put into `<template v-else>`
Draft:

```
<template>
  <component
    :is="tag"
    class="ui-list"
  >
    <template v-if="hasLoader">
      <component
        :is="loader"
        :is-loading="isLoading"
        v-bind="defaultProps.loaderAttrs"
      >
        <template #loader>
          <!-- @slot Use this slot to replace loader. -->
          <slot name="loader" />
        </template>
      </component>
    </template>
    <template v-else>
      <!-- @slot Use this slot to place list items. -->
      <slot>
        <template
          v-for="(item, key) in itemsToRender"
          :key="key"
        >
          <UiListItem
            v-bind="item"
          >
            <!-- Allow to use UiListItemSlots / BEGIN -->
            <template
              v-for="(_, name) in $slots"
              #[name]="data"
            >
              <slot
                v-bind="data"
                :name="name"
              />
            </template>
            <!-- END -->
            <!-- @slot Use this slot to replace list item content -->
            <slot
              :name="item?.name"
              v-bind="item"
            >
              <UiListItemContent :item="item" />
            </slot>
          </UiListItem>
        </template>
        <!-- @slot Use this slot to replace button -->
        <UiListItemButton v-if="hasButton">
          <template
            v-for="(_, name) in itemsToRender"
            #[name]="data"
          >
            <slot
              v-bind="data"
              :name="name"
            />
          </template>
        </UiListItemButton>
      </slot>
    </template>
  </component>
</template>
````